app/assets/stylesheets/tinder.scss

```diff
@@ -0,0 +1,112 @@
+#tinderslide {
+  width: 1000px;
+  height: 550px;
+  margin: auto;
+  background-color: $background;
+  margin-top: 100px;
+
+  img {
+    float: left;
+    margin: 20px 20px 0 20px;
+    border-radius: 10px;
+
+    -webkit-box-shadow: 2px 2px 15px 2px #a2a2a2;
+    -moz-box-shadow: 2px 2px 15px 2px #a2a2a2;
+    box-shadow: 2px 2px 15px 2px #a2a2a2;
+  }
+
+  li {
+    box-shadow: 0 0 15px rgba(0, 0, 0, 0.04);
+    padding: 0 0 30px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 0 30px` instead of `0 0 30px 0`